srml/treasury/src/lib.rs

```diff
@@ -311,6 +320,7 @@ impl<T: Trait> Module<T> {
 			Self::deposit_event(RawEvent::Burnt(burn))
 		}
 
+		// Budget doesn't include existential deposit thus the account must stay alive.
```

Comment:
So we create the treasury account once with ED and with `KeepAlive` we argue that it will never go beyond ED; qed

right?

Reply:
yes the buget to spend is balance - ED so even if the whole budget is spend then this doesn't go below ED. But indeed I'll rephrase a bit for clarification

Reply:
inline doc improved